server/controllers/AuthController.js

```diff
@@ -83,12 +87,17 @@ class AuthController {
             }
           });
         }
-        return res.status(401).json({
+        if (user.isVerified === false) {
+          return res.status(401).json({
+            message: 'Please click on the verification link that has been sent to your email'
+          });
+        }
+        return res.status(400).json({
           message: 'Username or password does not match.'
         });
       }
       return res.status(404).json({
-        message: 'Username or password does not match.'
+        message: 'You are yet Register. Kindly sign up.'
```

Comment:
correct this to `You are yet registered. Kindly sign up.`

Reply:
okay i will do that now